docs/content/en/docs/implementing/container.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Working with container runtime
+description: How to run tasks in container runtime
+weight: 95
+---
+
+Container runtime allows the Keptn Lifecycle Toolkit
+to execute tasks in Kubernetes containers as part of a Kubernetes Job.
+This is similar to the Keptn v1 Job Executor Service.
+
+This is a new feature introduced in v0.8.0
+that is useful for:
+
+- Running load/performance testing tools
+- Running tasks expressed in other languages such as Python
+- [TODO] What else?
+- Running SLO validations
+
+To implement this feature:
+
+- Define a
+  [KeptnTaskDefinition](../yaml-crd-ref/taskdefinition.md)
+  resource that defines the container
+- Populate a [KeptnApp](../yaml-crd-ref/app.md)
+  resource that associates that container definition
+  with the pre- and post-deployment tasks
+  that should run in it
+
+## Using containers for SLO validation
```

Comment:
This is more an evaluation, for that we have a separate resources -> KeptnEvaluation. I would maybe scratch this paragraph from here

Reply:
I found the KEP that mentioned this but it's obviously not something that is actually implemented and relevent.  I deleted it all.